### PR TITLE
Create asin.md

### PR DESCRIPTION
**Pull request by HKABIG**
created asin.md in accordance to the guidelines provided

### Description

created `docs/content/kotlin/concepts/math-methods/terms/asin/asin.md` entry for #3514 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/asin/asin.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The **`asin()`** function returns the arcsine of a number.
+
+## Syntax
+
+```pseudo
+asin(x)
+```
+
+The function accepts one mandatory parameter `x`. It should be a numeric value in the range between `-1` and `1` (inclusive). The argument can be of `Double`, `Float`, or `Long` datatype.
```
Comment (avdhoottt) [inline content/kotlin/concepts/math-methods/terms/asin/asin.md:25]:
```suggestion
The function accepts one mandatory parameter `x`. It should be a numeric value in the range between `-1` and `1` (inclusive). The argument can be of `Double`, and `Float` datatype.
```
Long is a numeric data type, it doesn't make as much sense to use it as an argument for the asin() function.

File: content/kotlin/concepts/math-methods/terms/asin/asin.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The function returns the arcsine of the argument, which represents an angle in radians. The return value will be of the same type as the argument.
```
Comment (avdhoottt) [inline content/kotlin/concepts/math-methods/terms/asin/asin.md:27]:
```suggestion
It returns the arcsine of the argument, which represents an angle in radians. The return value will be of the same type as the argument.
```

File: content/kotlin/concepts/math-methods/terms/asin/asin.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Example
+
+The example below demonstrates how to use the method to return the arcsine of `0.5`.
```
Comment (avdhoottt) [inline content/kotlin/concepts/math-methods/terms/asin/asin.md:31]:
```suggestion
The example below demonstrates how to use the asin() method to return the arcsine of `0.5`.
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@HKABIG Firstly, creating a PR on the main branch is discouraged, Consider using a new branch. I've mentioned some changes, please correct them.

**Comment by HKABIG** [comment]
@avdhoottt applied the requested changes please review

**Review by avdhoottt — APPROVED** [review]
Ready for the next round of Review

**Review by SSwiniarski — APPROVED** [review]
@HKABIG Looks good.

**Pull request merged**